webpack.config.js

```diff
@@ -46,7 +46,7 @@ const config = {
 							loader: 'sass-loader',
 							query: {
 								includePaths: [ 'editor/assets/stylesheets' ],
-								data: '@import "variables"; @import "mixins";',
+								data: '@import "variables"; @import "mixins"; @import "animations";',
```

Comment:
We might consider creating a `common.scss` to import these files so we don't have to manage the Webpack configuration directly.